### ISSUE TITLE
Open *.piskel file on double click. Windows.

### ISSUE THREAD

**Issue by MaxEden** [juliandescottes/piskel#596]
Hi! Is it possible to make windows build open files on double click?
I believe windows sends the filename as a first argument.

**Comment by MaxEden** [comment]
Hi again!
I did a little research
https://github.com/nwjs/nw.js/wiki/Handling-files-and-arguments
The code should be something like this
```
     if(pskl.utils.Environment.detectNodeWebkit())
      {
          var gui = require('nw.gui');
          var args = gui.App.argv;
          if(args && args.length>0 && args[0].endsWith(".piskel"))
          {
            var file = args[0];
            if(file.toLowerCase().endsWith(".piskel"))
            {
              pskl.utils.PiskelFileUtils.loadFromFile(file, function (piskel) {
                pskl.app.piskelController.setPiskel(piskel);
                });
            }
          }
      }
```
I'm not sure where is the right place to wedge it in, though. 😁

**Comment by MaxEden** [comment]
During my tinkering I found a very strange issue. Whenever I pass the **real file name** as a parameter, the app doesn't launch (or crashes, I'm not sure since I haven't found any logs)
What can be the cause?

**Comment by juliandescottes** [comment]
Not sure, but if you used the snippet you mentioned in your previous comment, `file` is simply a string (the path to the file you want to load), while pskl.utils.PiskelFileUtils.loadFromFile expects an actual File object (https://developer.mozilla.org/en/docs/Web/API/File).

**Comment by MaxEden** [comment]
No, no. Sorry for not being clear. It's reproducible on the latest build without any modifications.
I tried 0.8.0 and 0.10.0
https://i.imgur.com/wkcNS5x.png

**Comment by MaxEden** [comment]
Looks like it's a known issue of nw.js 🙁  Bummer.
https://github.com/nwjs/nw.js/issues/5242

**Comment by juliandescottes** [comment]
Looks like this is getting resolved in nwjs side? https://github.com/nwjs/nw.js/issues/5242

Might be worth trying again with the next stable release of nwjs including this.

**Comment by MaxEden** [comment]
Yeah, I've seen the comment. I hope it is really fixed and not just closed :D
I'll try to implement the feature in spare time.

**Comment by blakebuthod** [comment]
Been wanting this feature as well! Looks like it might have been fixed on the nwjs side though I haven't been able to get my files to work...
